nipype/algorithms/mesh.py

```diff
@@ -22,9 +22,40 @@
 from ..interfaces.base import (BaseInterface, traits, TraitedSpec, File,
                                BaseInterfaceInputSpec)
 from warnings import warn
+
+
+have_tvtk = False
+
+try:
+    from tvtk.api import tvtk
```

Comment:
Yep. A fix: https://github.com/nipy/nipype/issues/1178#issuecomment-140833948
And a related question: https://github.com/nipy/nipype/issues/972#issuecomment-140835937